src/gen/graphviz.ts

```diff
@@ -149,9 +121,6 @@ function callsToString(
     }
   }
 
-  // Don't output a file if callSite doesn't have call given conditions' hierarchies
-  if (!callHierarchyRelations.length) return "";
-
```

Comment:
Output even if callSite doesn't have callees.
This is for when we use interactive call graphs, users might get rid of all children under the root node, but we still wanna show the root node to continue viewing.